tests/compiler/nonnullable.json

```diff
@@ -0,0 +1,4 @@
+{
+  "asc_flags": [
+  ]
+}
```

Comment:
nl

Reply:
See above it's missing in a lot of json files.

Reply:
Oh, now I see. Ok

Reply:
I think then all the other files are wrong, and it may be good to fix them in a separate PR as well :). Let's just fix it here for now?